vnext/Microsoft.ReactNative/ReactApplication.cpp

```diff
@@ -23,7 +24,16 @@ using namespace Windows::UI::Xaml::Navigation;
 
 namespace winrt::Microsoft::ReactNative::implementation {
 
-ReactApplication::ReactApplication() noexcept {
+ReactApplication::ReactApplication() = default;
+
+ReactApplication::ReactApplication(IInspectable const &outer) noexcept : ReactApplication{} {
+  // The factory is usually called in the base generated class. We call it here to pass correct
+  // 'outer' interface to enable inheritance from the ReactApplication class in user code.
+  impl::call_factory<xaml::Application, xaml::IApplicationFactory>([&](xaml::IApplicationFactory const &f) {
```

Comment:
>xaml [](start = 21, length = 4)

These are added as part of WinUI3 change.
I think we can remove them in this change or replace with Windows::UI::Xaml::Application.

Reply:
My plan was to replace with windows::UI::Xaml::Application.

Reply:
sounds good!

---
In reply to: [421041603](https://github.com/microsoft/react-native-windows/pull/4806#discussion_r421041603) [](ancestors = 421041603)